client/src/main/scala/org/http4s/client/Client.scala

```diff
@@ -65,9 +58,7 @@ trait Client[F[_]] {
     * responsibility of callers of this service to run the response
     * body to dispose of the underlying HTTP connection.
     *
-    * This is intended for use in proxy servers.  `fetch`, `fetchAs`,
-    * [[toKleisli]], and [[streaming]] are safer alternatives, as their
-    * signatures guarantee disposal of the HTTP connection.
+    * This is intended for use in proxy servers.
```

Comment:
I would keep a weaker form of the deleted text.  "as they allow the client to manage resources without forcing the caller to consume the body of each returned response."